support/util/imagemetadata_test.go

```diff
@@ -146,7 +150,9 @@ func TestGetManifest(t *testing.T) {
 			}
 
 			if tc.validateCache {
-				_, exists := manifestsCache.Get(tc.expectedDigest)
+				parsedImageRef, err := reference.Parse(tc.imageRef)
+				g.Expect(err).NotTo(HaveOccurred())
+				_, exists := manifestsCache.Get(parsedImageRef.String())
 				g.Expect(exists).To(BeTrue())
 			}
```

Comment:
I'm thinking since we're always checking the cache now, can we also when `validateCache` is false that the image is not cached